pkgs/applications/window-managers/yabar/build.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchFromGitHub, cairo, gdk_pixbuf, libconfig, pango, pkgconfig
+, xcbutilwm, alsaLib, wirelesstools, asciidoc, libxslt, makeWrapper
+, configFile ? null, lib
+, rev, sha256, version
+}:
+
+stdenv.mkDerivation {
+  name = "yabar-${version}";
+
+  src = fetchFromGitHub {
+    inherit rev sha256;
+
+    owner = "geommer";
+    repo  = "yabar";
+  };
+
+  buildInputs = [
+    cairo gdk_pixbuf libconfig pango pkgconfig xcbutilwm
+    alsaLib wirelesstools asciidoc libxslt makeWrapper
+  ];
+
+  postPatch = ''
+    substituteInPlace ./Makefile \
+      --replace "\$(shell git describe)" "${version}" \
+      --replace "a2x" "${asciidoc}/bin/a2x --no-xmllint"
```

Comment:
Just as an aside, since `asciidoc` is in `buildInputs` it's bin directory will be in PATH. So it would be sufficient to have
```
--replace "a2x" "a2x --no-xmllint"
```
here. No need to change unless you feel like it. Just thought I'd highlight it.